<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>936</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>textparser UI</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QSplitter" name="horizontalSplitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QSplitter" name="verticalSplitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTreeWidget" name="astTreeWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <column>
        <property name="text">
         <string notr="true">AST tree</string>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
       </column>
      </widget>
      <widget class="QPlainTextEdit" name="plainTextEdit">
       <property name="font">
        <font>
         <family>Monospace</family>
         <pointsize>12</pointsize>
        </font>
       </property>
      </widget>
     </widget>
     <widget class="QPlainTextEdit" name="parserDefinitionWidget">
      <property name="font">
       <font>
        <family>Noto Sans Mono</family>
        <pointsize>18</pointsize>
       </font>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
